geode-core/src/main/java/org/apache/geode/management/internal/MBeanJMXAdapter.java

```diff
@@ -184,7 +187,16 @@ public void unregisterMBean(ObjectName objectName) {
       if (localGemFireMBean.get(objectName) != null) {
         localGemFireMBean.remove(objectName);
       }
-    } catch (NullPointerException | MBeanRegistrationException | InstanceNotFoundException e) {
+    } catch (InstanceNotFoundException instanceNotFoundException) {
```

Comment:
Not sure if this is the right place to make the change.

Reply:
Good point - I'm not sure that registerMBean is subject to the same race condition (multiple threads registering a given MBean), but for symmetry I see the argument that we should handle InstanceNotFoundExceptions there as well.  I will add that.

Reply:
Regarding the right place to make the change, after reading through the MBeanServer Javadocs, it appears that these exceptions are thrown either when an MBean is already registered when registering or unregistered when unregistering.  There is nothing that the callers would do with this information and it is essentially a no-op.  I think handling this exception as close to where it was thrown as possible is the right approach.